src/renderer/components/dialog/publicationInfos/PublicationInfo.tsx

```diff
@@ -193,6 +200,16 @@ class PublicationInfo extends React.Component<IProps, IState> {
                         </p>
                     </div>
                 </div>
+            </>;
+
+        return (
+            <Dialog open={true} close={this.props.closeDialog}>
+                {this.state.coverZoom ?
+                    <Cover publication={publication} coverTypeUrl="coverUrl" onclick={() => this.setState({
```

Comment:
Can we have additional CSS to ensure that the full cover image is displayed in a larger viewport than the thumbnail?
See screenshots:
![ThoriumThumbnailCoverImage](https://user-images.githubusercontent.com/625201/67314612-e1dffe80-f4fc-11e9-81d9-c9c1a9c92bb4.png)
![ThoriumFullCoverImage](https://user-images.githubusercontent.com/625201/67314616-e4425880-f4fc-11e9-8560-c58bc4641611.png)

Reply:
I don't have the skill on CSS, I don't know how to do this

Reply:
Cf PR comment on the top

Reply:
finally i found a little fix, just a constraint on the container